app/views/examples/example_radios_checkboxes.html

```diff
@@ -64,15 +62,13 @@ <h1 class="heading-large">
           </fieldset>
         </div>
 
-        <h1 class="heading-large">
-          Where should we send your new<br> passport?
-        </h1>
-
         <div class="form-group">
           <fieldset>
 
-            <legend class="visually-hidden">
-              Where should we send your new passport?
+            <legend>
+              <h1 class="heading-large">
+                Where should we send your new passport?
```

Comment:
I have removed it because I didn't understand why it was there in the first place. I think it's generally bad practice to have `<br>`s in a heading. I tried to find out via `git blame` why it was there but got lost trying to get before October 2014.

BTW, without this change, it currently looks broken on mobile because of that `br`:
![br-in-h1](https://user-images.githubusercontent.com/108893/27647714-22779f30-5c24-11e7-8eb8-3616db799506.png)